assets/js/stripe-admin.js

```diff
@@ -86,6 +88,25 @@ jQuery( function( $ ) {
 					} );
 				} );
 			} );
+
+			// Webhook verification checks for timestamp within 5 minutes so warn if
+			// server time is off from browser time by > 4 minutes.
+			var timeDifference = Date.now() / 1000 - wc_stripe_settings_params.time;
+			var isTimeOutOfSync = Math.abs( timeDifference ) > 4 * 60;
+			$( '#woocommerce_stripe_test_webhook_secret, #woocommerce_stripe_webhook_secret' )
+				.on( 'change input', function() {
```

Comment:
Why wait to change input?

Reply:
It doesn't actually wait – see `.change()` call below. Also, the `$( this ).val()` condition below makes sure the secret is not empty.

Reply:
Ah - missed that!